i18n/ja/docusaurus-plugin-content-docs-user-manual/current/project-and-workspace/project/overview.mdx

```diff
@@ -3,88 +3,90 @@ title: 概要
 sidebar_position: 1
 ---
 
-これは日本語
-
-Projects are where all the magic happens in Re:Earth. Edit, publish, import your data and more.
+プロジェクトでは、編集・公開・データのインポートなどが出来ます。
 
 <Image path={require('./img/1-1.png').default} width="60%"/>
 
-Once you create a project you can open the editor page and by default you will see a digital earth floating in the universe. When a project is created, a scene will be created, attached and used by your project. Currently, only Cesium-based scenes are available, but more will come in the future. For more information about scenes, please see the chapter on [scenes](/user-manual/scene/ideas-of-scene).
+プロジェクトを作成すると、編集画面を開くことができます。
+編集画面には、宇宙に浮かぶデジタル地球儀が表示されています。Re:Earthではこれをシーンと呼んでいます。
+現在、表示可能なシーンはCesiumのみですが、今後様々なシーンに対応する予定です。
+シーンについての詳しい情報は、以下の章をご覧ください。
 
 <Image path={require('./img/1-2.png').default} />
 
-As described in the previous chapters, you can add and build various content on the earth. Treat the project like an easle, the earth itself the canvas, and Re:Earth the set of brushes and tools to help you create your vision.
+先に述べたように、地球上には様々なコンテンツを追加・作成することができます。プロジェクトは「画架」、地球は「キャンバス」、Re:Earthは「ブラシ」と「ツール」と考えて、あなたのイメージを膨らませていきましょう。
 
-The following operations are available on a project:
+プロジェクトでは以下の操作が可能です：
 
-- [Create](/user-manual/project-and-workspace/project/create-a-new-project)
-- [Preview]
-- [Publish]
-- [Archive](/user-manual/project-and-workspace/project/settings-archiving-and-deleting)
-- [Delete](/user-manual/project-and-workspace/project/settings-archiving-and-deleting)
+- [作成](ja/user-manual/project-and-workspace/project/create-a-new-project)
```

Comment:
please don't add "ja" in front of the link, the system know it is on "/js", which is automatically。